Dockerfile.rhtap

```diff
@@ -1,21 +1,38 @@
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.22 as builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23.6 AS builder
 
-# Run this with docker build --build_arg $(go env GOPROXY) to override the goproxy
+WORKDIR /workspace
+
+# Run this with docker build --build-arg goproxy=$(go env GOPROXY) to override the goproxy
 ARG goproxy=https://proxy.golang.org
 ENV GOPROXY=$goproxy
-ENV GOFLAGS="-mod=readonly"
 
-WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+
+# Cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+#RUN --mount=type=cache,target=/go/pkg/mod \
+#    go mod download
```

Comment:
looks like this is done in lines 22-23 below and this block can be removed.

Reply:
Thank you @orenc1  Removed.